atd-vzv/src/views/map/Map.js

```diff
@@ -101,6 +104,7 @@ const Map = () => {
       filters,
       dateRange,
       mapPolygon,
+      mapRequestFields,
```

Comment:
I added a parameter to the helper that builds up the [Socrata $select parameter](https://dev.socrata.com/docs/queries/select.html) so we can control what fields are returned when this function is reused. We were requesting unneeded geo fields here previously.